### ISSUE TITLE
Plugin breaks when there is a title without a link in SUMMARY.md

### ISSUE THREAD

**Issue by tianshuo** [todvora/gitbook-plugin-image-captions#15]
If there is an item in SUMMARY.md without a link, the plugin breaks by outputting:

```
Error: EISDIR: illegal operation on a directory, read
```

**Comment by tianshuo** [comment]
# To reproduce this bug

Create a `SUMMARY.md` file like this:

```
# Summary
* intro
```

# Fix

I have added an `if` to check that the link exists in this PR #16

**Comment by todvora** [comment]
Fixed in #16, closing now. Thank you very much!
